### PR TITLE
Ft 136 remove icons from components and merge to icons

### PR DESCRIPTION
**Pull request by MightyDevKnight**

### CODE REVIEW

File: -L [hunk 1]
```diff
@@ -0,0 +1 @@
+Redirecting to https://classic.yarnpkg.com/install.sh
```
Comment (six7) [inline -L:1]: Is this required?

File: src/app/components/Icon.tsx [hunk 2]
```diff
@@ -1,25 +1,25 @@
 import React from 'react';
```
Comment (six7) [inline src/app/components/Icon.tsx:1]: Is this file still required? I think we can remove it and if there's imports left change those to svg?

File: src/app/components/ThemeSelector/ThemeSelector.tsx [hunk 3]
```diff
@@ -15,7 +15,7 @@ import {
 import { Flex } from '../Flex';
 import Text from '../Text';
 import { styled } from '@/stitches.config';
-import { IconToggleableDisclosure } from '../icons/IconToggleableDisclosure';
+import { IconToggleableDisclosure } from '@/icons/IconToggleableDisclosure';
```
Comment (six7) [inline src/app/components/ThemeSelector/ThemeSelector.tsx:18]: Can we change this one to .svg as well?

**Review by six7 — CHANGES_REQUESTED** [review]

File: -L (hunk 1 above)
Comment (MightyDevKnight) [reply -L:1]: removed

File: src/app/components/ThemeSelector/ThemeSelector.tsx (hunk 3 above)
Comment (MightyDevKnight) [reply src/app/components/ThemeSelector/ThemeSelector.tsx:18]: This icon component has props. 
 {activeTokenSet}
<IconToggleableDisclosure open={tokenSetsVisible} />

File: src/app/components/Icon.tsx (hunk 2 above)
Comment (MightyDevKnight) [reply src/app/components/Icon.tsx]: removed

File: src/app/components/ThemeSelector/ThemeSelector.tsx (hunk 3 above)
Comment (six7) [reply src/app/components/ThemeSelector/ThemeSelector.tsx:18]: hm, ok. for now it's fine but i'll create a followup ticket to solve this better. icons should really only be .svg and if there's a transformation happening it should be a wrapping component using that icon

**Review by six7 — APPROVED** [review]
